base/array.jl

```diff
@@ -320,6 +320,8 @@ oneunit(x::AbstractMatrix{T}) where {T} = _one(oneunit(T), x)
 
 convert(::Type{Vector}, x::AbstractVector{T}) where {T} = convert(Vector{T}, x)
 convert(::Type{Matrix}, x::AbstractMatrix{T}) where {T} = convert(Matrix{T}, x)
+convert(::Type{Matrix}, x::Vector{T}) where {T} = Matrix(reshape(x, :, 1))
+convert(::Type{Matrix}, x::RowVector{T,S}) where {T,S} = Matrix(reshape(x, 1, :))
```

Comment:
These should probably be constructors instead of converts since I don't think we want these to happen implicitly. Also I think these should not use `reshape` since the vector will not be resizeable afterwards.